src/core_plugins/kibana/public/dashboard/top_nav/clone_modal.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiModal,
+  KuiModalHeader,
+  KuiModalHeaderTitle,
+  KuiModalBody,
+  KuiModalBodyText,
+  KuiModalFooter,
+  KuiButton,
+  KuiModalOverlay
+} from 'ui_framework/components';
+
+export class DashboardCloneModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      newDashboardName: props.title
+    };
+  }
+
+  componentDidMount() {
+    this.confirmButton.focus();
```

Comment:
We officially support these browsers: https://www.elastic.co/support/matrix#show_browsers

No need to optimize for mobile browsers, though Kibana should really still *function* in those browsers, ideally.